examples/plot_simulate_gamma.py

```diff
@@ -31,6 +31,20 @@
 # Now let's simulate the dipole
 
 net = Network(params)
+
+# add a tonic Poisson-distributed excitation to pyramidal cells
+weights_ampa = {'L2_pyramidal': 0.0008, 'L5_pyramidal': 0.0075}
+synaptic_delays = {'L2_pyramidal': 0.1, 'L5_pyramidal': 1.0}
+rate_constants = {'L2_pyramidal': 140.0, 'L5_pyramidal': 40.0}
+# XXX online docs had a seed of -3 (!). This worked because extpois was added
+# last, the first artificial cell having gid=1352, leading to seeds >= 1349
+# With the new API, seeds begin from 270, so init_seed = 1352 - 3 - 270
```

Comment:
yes, negative seed was bogus

same comment as above. Remove this from example. Perhaps we can advice users how to set seeds in `whats_new.rst` under "API" section? If they were using HNN-core before and want to achieve the same results as before. I predict this will be a small fraction of people though